todoapp/src/components/toDoList.css

```diff
@@ -0,0 +1,23 @@
+.imageAllignment {

```

Comment:
make responsive according to each screen size

Reply:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/102802548/217458247-0ef2a4cd-a9aa-4202-839c-603ea28a99af.png">

adding scroll in mobile view